src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -784,6 +784,11 @@ public ActivityTraceId TraceId
         /// </summary>
         public bool IsAllDataRequested { get; set;}
 
+        /// <summary>
+        /// Indicate if the this Activity object is stopped
+        /// </summary>
+        public bool IsStopped { get => IsFinished; }
```

Comment:
Why not just rename IsFinished to IsStopped?

Reply:
No reason other than less code change :-) I'll apply this suggestion.